packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -206,13 +206,23 @@ export class DataSyncPlugin extends GraphbackPlugin {
   }
 
   private getDeltaTypeFieldNames(modelTC: GraphQLObjectType): string[] {
-    const DeltaTypeFieldEntries = Object.entries(modelTC.getFields()).filter((e: [string, GraphQLField<any, any>]) => {
-      // Remove relationship fields from delta Type
-      return parseRelationshipAnnotation(e[1].description) === undefined;
-    });
 
-    return DeltaTypeFieldEntries.map(([fieldName, _]: [string, GraphQLField<any, any>]) => {
-      return fieldName
-    });
+    return Object.entries(modelTC.getFields()).map(([fieldName, field]: [string, GraphQLField<any, any>]) => {
+      const relationship = parseRelationshipAnnotation(field.description);
+      if (relationship) {
+        if (relationship.kind === "oneToOne") {
+          return relationship.key;
+        }
+        if (relationship.kind === "manyToOne") {
+          // This seems to be always set for manyToOne
```

Comment:
Probably because manyToOne is always a generated field, and we always set "key".